### ISSUE TITLE
XMR-STAK cpu -enable cryptonight_r asm 'amd_avx' cpu's-

### ISSUE THREAD

**Issue by seraphsatan** [fireice-uk/xmr-stak#2307]
Please provide as much as possible information to reproduce the issue.

# Basic information
  - Type of the CPU. AMD r7 1800x
  - Type of the GPU (if you try to miner with the GPU).

# Compile issues
  - Which OS do you use?  Windows 10
  ```
 # Stability issue
  - Is the CPU or GPU overclocked? yes 4.0Ghz
  - Is the Main memory of the CPU or GPU undervolted?
[cpu.txt](https://github.com/fireice-uk/xmr-stak/files/2949631/cpu.txt)

"cpu_threads_conf" :
[
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 0 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 2 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 4 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 6 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 8 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 10 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 12 },
    { "low_power_mode" : false, "no_prefetch" : true, "asm" : "amd_avx", "affine_to_cpu" : 14 },
],

When the output informs of the cryptonight_r asm issue the hashrate drops to ~20% of max for a while (unsure of duration, but a minute or more as the 60s designation on the report shows 17 rather than the normal ~72) then back to normal.

**Comment by psychocrypt** [comment]
could you please post the output that I know what you mean with the asm
issue.

**Comment by seraphsatan** [comment]
![cryptonight](https://user-images.githubusercontent.com/48412068/54090202-f2fe4580-4347-11e9-83db-11fb7182d5d7.png)
Hope that helps and is what you were looking for.

Thx for the fast reply.  I am impressed.

**Comment by psychocrypt** [comment]
slowdown fixed in #2314. The fix will be available  soon in the next release
